src/from_csv.rs

```diff
@@ -97,8 +97,8 @@ lazy_static::lazy_static! {
 #[test]
 fn num_unwrapper() {
     assert_eq!(NUM_UNWRAPPER.replace_all("Decimal('0')", "$n"), "0");
-    assert_eq!(NUM_UNWRAPPER.replace_all("Decimal('0.0')", "$n"), "0");
-    assert_eq!(NUM_UNWRAPPER.replace_all("Decimal('-0.0')", "$n"), "0");
+    assert_eq!(NUM_UNWRAPPER.replace_all("Decimal('0.0')", "$n"), "0.0");
```

Comment:
Can you explain more what this does?

Reply:
oh god, I have to explain cursed data migration scripts I wrote ... :sweat: 
I used a tool to convert the entire dynamodb into a massive csv file, you can find it here:
https://pypi.org/project/export-dynamodb/
Unfortunately, for data nested beyond a single row, you get some sort of weird JSON thing, but actual numbers aren't used, they're wrapped in `Decimal` because of dynamodb's special "not json" format (that's basically json), so I have this cursed regex to quickly yank the numbers out of their decimal wrappers.